abiquo_inventory.py

```diff
@@ -232,6 +232,16 @@ def get_vms(self):
             self.fail_with_error(e)
         return vms
 
+    def get_vms_by_vdc(self, vdc:str):
+        code, vms = self.api.cloud.virtualdatacenters(vdc).action.virtualmachines.get(
```

Comment:
https://wiki.abiquo.com/api/latest/VirtualDatacentersResource.html#list-virtual-machines-in-a-virtual-datacenter
not sure if the lib is handling pagination, just make sure its able to get more than 25 vm.
If not implemented, consider adding the ability to iterate pages or (some performance concerns) fetch all using query param ```limit=0```

Reply:
As far as I see, the python library has no pagination implemented. I will add this limit=0, but I agree with you it's not the best solution.

Reply:
@vicentgodella, if you're using the abiquo python client lib, it actually handles pagination: https://github.com/abiquo/api-python-client/blob/master/abiquo/client.py#L145

Reply:
Yes, I found it a while ago :smile: 
I just removed the `limit` parameter.

Thanks for your comment!